### ISSUE TITLE
Time is created in UTC with local time

### ISSUE THREAD

**Issue by thiagodiniz** [rails/rails#3145]
When a Model is created using the mass assignment, and the model has a time attribute.

The Time attribute is created in local time but with UTC time zone even with config.time_zone properly setted.

I was studying this wrong behavior and guess that the error is here:

https://github.com/rails/rails/blob/master/activerecord/lib/active_record/attribute_methods/time_zone_conversion.rb#L62

As you can see, in the line 62 it only consider :datetime and :timestamp. it does not consider :time to be timezone aware.

**Comment by zouis** [comment]
That's not working for me. I updated to 4.2, set config.time_zone = 'Europe/Paris' but still have UTC time   2000-01-01 18:00:00 UTC
Am I missing something ?

**Comment by MrOvitch** [comment]
2 years later, we have this problem (only in local/staging env but not in production..)
we have an old app that uses : 
- ruby '2.1.5'
- gem 'rails', '3.2.14'

applications.rb have
config.time_zone = 'Europe/Paris' 

without config.active_record.default_timezone
only, without config.active_record.default_timezone

if we add config.active_record.default_timezone = :local
time are stored utc+2 but displayed utc ...

What can we do in order to fix it for rails 3.2.14 ? (without uppgrading to rails >4.3)

**Comment by paulomcnally** [comment]
Rails 5.1.4 

ApplicationController.rb
```
...
around_action :settings_time_zone
...

...
def settings_time_zone(&block)
    puts @settings[:time_zone]
    Time.use_zone('America/Managua', &block)
end
...
```

*time_select* ignore the timezone.
